tests/items/profile_test.py

```diff
@@ -46,7 +46,7 @@ def test_to_dict(cobbler_api, cleanup_to_dict):
     result = profile.to_dict()
 
     # Assert
-    assert len(result) == 44
+    assert len(result) == 43
```

Comment:
The test is the same. It depends on how many attributes are stored in the dict, that depends on the `Profile` class, not on the test itself.

If we check the main branch now, it's at `45`. In my opinion this is not a good way to do such tests because it encourages just de-/incrementing the number when a test fails. This number could be computed if we moved the set of ignored attributes somewhere accessible (it's currently only inline in `Item.__is_dict_key`).